README

```diff
@@ -1,3 +1,55 @@
-This is the unit-test for ACRN project. We have two type of unit-test:
- - guest side. The unit-test will be run as a ACRN guest
- - host side. The uni-test will be run at ACRN host side.
+# Readme
+  This is the unit-test for ACRN project. The unit-test will run as a ACRN guest.
+  Notes: acrn-unit-test now is still test framework ready only, no much detailed test case yet. 
+         But you can try and add new cases according to the following descriptions.
+
+# acrn-unit-test directory structure.
+    ./guest:			configure script, top-level Makefile
+    ./guest/scripts:		helper scripts for building and running tests
+    ./guest/lib:		general architecture neutral services for the tests
+    ./guest/lib/x86:		architecture dependent services for the tests
+    ./guest/x86:		the sources of the tests and the created objects/images
+
+# acrn-unit-test building steps.
+  1.Get unit-test code
+  $git clone https://github.com/projectacrn/acrn-unit-test.git  # branch: master
+
+  2:Enter acrn-unit-test/guest, config and compile. This directory contains sources for ACRN test suite.
+  To create all test images do:
+
+    ./configure --arch=x86_64
```

Comment:
```suggestion
    $ ./configure --arch=x86_64
```

Be consistent: if we use a prompt in one command, use it everywhere. Same for all command in the rest of this file.

Reply:
My preference would be that we do **not** use the command prompt. I prevents people from doing a copy-paste of the instructions.